scripts/distTemplate.js

```diff
@@ -0,0 +1,11 @@
+import * as allExports from '../src/index';
+
+(function (root) {
+  const LogWeasel = Object.keys(allExports).reduce((key, obj) => {
+    obj[key] = allExports[key];
+    return obj;
+  }, {});
+
+  root.LogWeasel = LogWeasel;
```

Comment:
Is the idea that the minified script would be loaded as a script in an app's head; the script would then attach LogWeasel to the window object? Would engineers call LogWeasel manually, or does LW execute the global automatically?